src/util/parse.js

```diff
@@ -23,15 +19,54 @@ export const parse = async (options) => {
     options.flavour = options.flavor;
   }
 
-  options.srcDir = options.srcDir ?? undefined;
-  options.version = options.version ?? undefined;
-  options.flavour = options.flavour ?? undefined;
-  options.platform = options.platform ?? undefined;
-  options.arch = options.arch ?? undefined;
-  options.outDir = options.outDir ?? undefined;
+  options.srcDir = options.srcDir ?? "./";
+  options.mode = options.mode ?? "build";
+  options.version = options.version ?? "latest";
+  options.flavour = options.flavour ?? "sdk";
```

Comment:
`'normal'` would likely be the desired default flavor for a build in almost all circumstances.

Cases I can think of where I'd want SDK:

1. If I did a "Stable" and "Beta" or "Canary" release of an app, the beta/canary version may ship with dev tools enabled, while the "Stable" wouldn't.
1. I might do an sdk build locally manually to debug, check, or validate something with the build, though this would be fairly rare.
1. Some apps are specifically aimed at developers, (like Scout-App), and want to encourage tinkering with the app itself.

I can't really think of any other scenario. Most NW.js app creators just ship the 'normal' version.

```suggestion
  options.flavour = options.flavour ?? "normal";
```
If this default is changed, the documentation table in `docs/index.md` would need updated as well.

Reply:
Also, most English speaking countries follow the America variations for `color` and `flavor`, as opposed to the British spelling. I'm actually fine if the API supports both. But specifically all documentation and API for NW.js uses the American style.

* https://nwjs.io/versions.json
* `console.log(nw.process.versions['nw-flavor'])`
* https://nwjs.readthedocs.io/en/latest/For%20Users/Advanced/Build%20Flavors/#build-flavors

```suggestion
  options.flavour = options.flavour || options.flavor || "normal";
```

Also changed to OR, since the flavor can only be normal or sdk, so no reason to allow passing in other falsy values with nullish co.

Reply:
Agree with `normal` as default flavour. Leaning towards keeping it American English. I spelt it as `flavour` out of habit but keeping it consistent makes more sense.